### PR TITLE
41

### PR DESCRIPTION
**Pull request by osaosula**
Assign Clinician UI

### CODE REVIEW

File: src/pages/AssignClinician.jsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+} from "@mui/material";
+import Grid from "@mui/material/Grid2";
+import { useQuery } from "@tanstack/react-query";
+import { useState } from "react";
+import { getEmployeesByType } from "../services/employees";
+const AssignClinician = ({ open, onClose }) => {
+  const [clinician, setClinician] = useState("");
+  const { isPending, data: clinicians } = useQuery({
+    queryKey: ["clinicians"],
+    queryFn: getEmployeesByType("cinician"),
```
Comment (BorisLaboratory) [inline src/pages/AssignClinician.jsx:19]: Looks like a typo "cinician"

**Review by BorisLaboratory — CHANGES_REQUESTED** [review]
Looks good but some observation. See comment

File: src/pages/AssignClinician.jsx (hunk 1 above)
Comment (osaosula) [reply src/pages/AssignClinician.jsx]: Fixed

**Review by BorisLaboratory — APPROVED** [review]
Looks good

**Pull request merged**